files/wheelnav/update.json

```diff
@@ -0,0 +1,8 @@
+{
+  "packageManager": "github",
+  "name": "wheelnav",
+  "repo": "softwaretailoring/wheelnav",
+  "files": {
+    "include": ["wheelnav.js", "wheelnav.min.js", "wheelnav.min.js.map"]
```

Comment:
I take it already now, I found this page: https://github.com/jsdelivr/libgrabber#add-updatejson-schema
The update.json fixed.